video/vaapi.c

```diff
@@ -264,30 +238,23 @@ static void drm_destroy(void *native_ctx)
     talloc_free(ctx);
 }
 
-static void drm_create(VADisplay **out_display, void **out_native_ctx)
+static void drm_create(VADisplay **out_display, void **out_native_ctx,
+                       const char *path)
 {
-    static const char *drm_device_paths[] = {
-        "/dev/dri/renderD128",
-        "/dev/dri/card0",
```

Comment:
the loss of this fallback device still disturbs me. this was discussed in the previous review, but I wasn't terribly satisfied with the answer. why was it here in the first place?

Reply:
/dev/dri/cardX is legacy interface which right now isn't even recognized by VAAPI as a valid handle, probably because it lacks some features the new renderer interface offers. Considering we dropped support for old VAAPI versions and the fact that the card interface is no longer functioning as a fallback I think its okay.
What previous review?